### ISSUE TITLE
Currently not working

### ISSUE THREAD

**Issue by TheOfficialBarn** [asrvd/lyrist#9]
For all songs I can think of, the API currently is failing to fetch and display lyrics. I use the API in [TuahTone](https://tuahtone.vercel.app/) and noticed that my website can't find lyrics anymore.

**Comment by pawelzet** [comment]
Yup, I can confirm that it's not working at all. "Lyrics not found".

**Comment by asrvd** [comment]
yes, something's wrong with the library i'm using to fetch the lyrics, trying to fix.

**Comment by SeydEf** [comment]
Did the problem get solved?

**Comment by pawelzet** [comment]
Hi, is there any update on that?

**Comment by SeydEf** [comment]
We are waiting for you brother 🥲

**Comment by TheOfficialBarn** [comment]
Please 🥺

I don't want to change my code 😭

**Comment by asrvd** [comment]
hey everyone, sorry for the delay, the api is fixed again and i have added ratelimits (50 req per hour), i was really tired of people misusing the api and using up my vercel usage quota, please self-host if you need better increased usage, thankyou.

**Comment by asrvd** [comment]
reopening as the lyrics fetching still doesnt seem to work, its working on local env but failing on prod (returns empty object), would be nice if someone can look into it, ill try to fix soon

**Comment by dharmiknotop** [comment]
hey @asrvd , were you able to fix it? I am also facing the same issue
